README.md

```diff
@@ -69,5 +86,7 @@ UserApp.add_all_users = function(data) {
 };
 ```
 
+There's a comma after get. It should be removed.
```

Comment:
Nope, that's not the problem. The `UserApp.add_all_users(data);` line needs to be inside of the `.done()` callback in order to have access to `data`.